rllib/agents/ppo/ppo.py

```diff
@@ -185,12 +183,12 @@ def validate_config(config):
         logger.warning(
             "Using the simple minibatch optimizer. This will significantly "
             "reduce performance, consider simple_optimizer=False.")
-    elif config["use_pytorch"] or (tf and tf.executing_eagerly()):
+    elif config["framework"] != "torch":
```

Comment:
Shouldn't this be `config["framework"] in ["torch", "tfe"]`?

Reply:
Good catch. I probably meant `!= "tf"`.

Reply:
done